cluster/gce/upgrade.sh

```diff
@@ -102,6 +100,10 @@ function wait-for-master() {
 function prepare-upgrade() {
   ensure-temp-dir
   detect-project
+  # TODO(mbforbes): Do we still need get-password?
+  get-password
+  # TODO(mbforbes): Do we need get-bearer-token here?
```

Comment:
We shouldn't need auth data when updating the master, since the master stores the auth data on a PD which survives rebuilding the VM (and if we lose that data PD we are pretty well screwed anyway since it contains the cluster state). 

For the nodes, you are already scraping the kubelet token and kube proxy token from the existing metadata, so that can be used to fill in those two fields. 

Unless I'm reading this incorrectly, this is within upgrade.sh so it shouldn't affect cluster provisioning to remove these here.